token-syncer/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+log4j.rootLogger=DEBUG, stdout
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{4} - %m%n
+log4j.appender.stdout.Threshold=DEBUG
+
+# DEBUG is way too noisy for some categories
+log4j.logger.org.apache.http=INFO
```

Comment:
Needed?

Reply:
Reduced it to only contain `log4j.logger.org.eclipse.jetty`